Support for group into command_line auth provider (#27355)
home-assistant/home-assistant.github.io: source/_docs/authentication/providers.markdown

```txt
name = John Doe
group = system-users
local_only = true
```

Leading and trailing whitespace, as well as lines starting with `#` are ignored. The following variables are supported. More may be added in the future.

- `name`: The real name of the user to be displayed in their profile.

> - `group`: The user group, use value `system-admin` for administrator (this is the default) or `system-users` for regular users.
> - `local_only`: The user can only log in from the local network if you set the value to `true`. If you do not define this variable, the user will be able to log in from anywhere.

Reviewer: ```suggestion
- `group`: The user group uses the value `system-admin` for administrator (this is the default) or `system-users` for regular users.
- `local_only`: The user can only log in from the local network if you set the value to `true`. If you do not define this variable, the user can log in from anywhere.
```
